.gitignore

```diff
@@ -21,4 +21,5 @@ metastore_db/
 derby.log
 dependency-reduced-pom.xml
 rss-*.tgz
-
+hadoop-*.tar.gz
+/deploy/kubernetes/hadoopconfig/*
```

Comment:
We need a new line at the end of the file.
Maybe we don't need `hadoopconfig` directory.

Reply:
OK